.changes/unreleased/tinymce-TINY-8368-2024-02-09.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Fix non-public `TableSelectionChange` event to not use Sugar or Optional objects
```

Comment:
Should we even have a changelog if it's an internal thing? This kind of exposes it.

Reply:
Good point. I'll remove it err on the side of caution